### DISCUSSION TITLE
issues with ProxyPass and certificates in 2.0.12

### DISCUSSION THREAD

**Discussion by fkaddereit** [dropwizard/dropwizard#4764] [General]
I'm upgrading from 1.3.12 to 2.0.12 and got everything working fine. But once I put it in production where SSL is needed, it fails to work. Normally it works and looks like this:

``SSLProxyEngine On
ProxyPass / https://my.example.com:12333/``

The whole project runs on https://my.example.com/ . Now after the update as soon as a POST comes along the way, it redirects to https://my.example.com:12333/ and declares an unsafe page. When looking for the used certificate, it claims "localhost". As soon as I roll back to 1.3.12, everything works just fine again.

``server:
  applicationConnectors:
    - type: https
      port: 12333
      validateCerts: false``

That's the config side in the yml. What changed?

**Comment by joschi** [comment]
Check out the `useForwardedHeaders` setting for the HTTPS connector: https://www.dropwizard.io/en/latest/manual/configuration.html#connectors

Refs https://github.com/dropwizard/dropwizard/issues/3023

**Comment by fkaddereit** [comment]
worked, thxa lot!
